fe/fe-core/src/main/java/com/starrocks/common/proc/TablesProcDir.java

```diff
@@ -142,24 +144,24 @@ public ProcResult fetchResult() throws AnalysisException {
         return result;
     }
 
-    private long findReplicaCount(TableType tableType, Table table) {
-        if (tableType == TableType.OLAP) {
+    private long findReplicaCount(Table table) {
+        if (table.isNativeTable()) {
```

Comment:
Would the word `Native` be confused with `cloudNative` ?

Reply:
Native table includes olap table, materialized view, lake table.
https://github.com/StarRocks/starrocks/blob/main/fe/fe-core/src/main/java/com/starrocks/catalog/Table.java#L55

refer to bigquery: https://cloud.google.com/bigquery/docs/tables-intro